docs/mkdocs.yml

```diff
@@ -45,7 +45,7 @@ pages:
 - Datasets: datasets.md
 - Applications: applications.md
 - Backend: backend.md
-- Initializations: initializations.md
+- Initializations: initializers.md
```

Comment:
I just matched the names in autogen.py and mkdocs.yml. Do you have a link which works with "Initializers:"